osquery/tables/system/shell_history.cpp

```diff
@@ -23,7 +23,7 @@ namespace osquery {
 namespace tables {
 
 const std::vector<std::string> kShellHistoryFiles = {
-    ".bash_history", ".zsh_history", ".zhistory", ".history",
+    ".bash_history", ".zsh_history", ".zhistory", ".history",".sh_history",
```

Comment:
Just a small nit, no change needed, but there should be a space between the comma and start of the string literal.

If you run `make format` after `git add .` clang-format will apply the recommended styling. :)